docs/web_reference.rst

```diff
@@ -1784,9 +1784,15 @@ Application and Router
                               by default it's not allowed and HTTP/403 will
                               be returned on directory access.
 
-      :param bool follow_symlinks: flag for allowing to follow symlinks from
-                              a directory, by default it's not allowed and
-                              HTTP/404 will be returned on access.
+      :param bool follow_symlinks: flag for allowing to follow symlinks that lead
+                              outside the static root directory, by default it's not allowed and
+                              HTTP/404 will be returned on access.  Enabling ``follow_symlinks``
```

Comment:
How about putting most of this into a `.. danger::` block?

Reply:
Feel free to make an edit, I'm not sure how that would be formatted with the param syntax.

Reply:
As long as the indentation is kept, it should work. But I won't insist. It can get messy.